services/QuillLMS/app/views/pages/shared/_footer.html.erb

```diff
@@ -23,14 +25,35 @@
   <% else %>
     <footer class="<%=footer_class%>">
       <section aria-labelledby="premiumFooterBannerHeader" class="<%=banner_class%>">
-        <div class="left-side-container">
-          <p class="primary-header" id="premiumFooterBannerHeader">Go Premium, Boost Outcomes</p>
-          <p class="body">Teachers know the power of Quill’s immediate, targeted feedback in driving writing outcomes. Amplify those results with Quill Premium through custom reporting, professional learning, expanded rostering, and more.</p>
-          <div class="buttons-container">
-            <a class="quill-button small contained focus-on-light" href="/premium" rel="noopener noreferrer" target="">Explore Premium</a>
+        <div class="banner-card">
+          <div class="content-container">
+            <div class="text-container">
+              <h1>1.6x Rate of Growth</h1>
```

Comment:
I think I'd probably make these `<h2>`, since `<h1>` should be reserved for the main page's primary heading.